src/messages/en.ts

```diff
@@ -1,14 +1,6 @@
 import { Messages } from './types';
 
 export const messages: Messages = {
-  // The error messages on required fields
-  required: {
-    default: 'Please provide a response',
```

Comment:
Having centralized user-facing strings seems like a necessity for any kind of internationalization to take place.

Reply:
Agreed, we've created a ticket for this in our backlog.